core/src/main/scala/kafka/tools/ConsoleProducer.scala

```diff
@@ -146,62 +146,71 @@ object ConsoleProducer {
       .describedAs("size")
       .ofType(classOf[java.lang.Integer])
       .defaultsTo(200)
-    val messageSendMaxRetriesOpt = parser.accepts("message-send-max-retries", "Brokers can fail receiving the message for multiple reasons, and being unavailable transiently is just one of them. This property specifies the number of retries before the producer give up and drop this message.")
+    val messageSendMaxRetriesOpt = parser.accepts("message-send-max-retries", "Brokers can fail receiving the message for multiple reasons, " +
+      "and being unavailable transiently is just one of them. This property specifies the number of retries before the producer give up and drop this message. " +
+      "This is the option to control the `retries` in producer configs.")
```

Comment:
Add the last line to help users understand this option is mapping to which producer config. Same as below.